src/command/dev/next/router/config/parser.rs

```diff
@@ -161,8 +161,8 @@ health_check:
         );
         let config_yaml = serde_yaml::from_str(&config_yaml_str)?;
         let router_config = RouterConfigParser { yaml: &config_yaml };
-        let health_check = router_config.health_check();
-        assert_that!(health_check).is_equal_to(is_health_check_enabled);
+        let health_check = router_config.health_check_endpoint();
+        assert_that!(health_check.is_ok()).is_equal_to(is_health_check_enabled);
```

Comment:
Tried to fix up these tests because they seemed to have rotted. If this wasn't the intent of either test, let me know and I'll get them fixed up